lib/views/item/item-detail.js

```diff
@@ -95,15 +114,15 @@ class ItemDetail extends React.Component {
     );
   }
 
-  renderUrlBar({ url }) {
+  renderUrlBar({ url } = {}) {
     return (
       <View style={styles.urlBar}>
         <Text style={styles.url}>{url}</Text>
       </View>
     );
   }
 
-  renderText({ title, description, url, siteName = 'website' }) {
+  renderText({ title, description, url, siteName = 'website' } = {}) {
```

Comment:
Worth noting that it may not always be a 'website'. If you have the native app installed, it will open there.